authority-portal-frontend/src/app/pages/authority-organization-detail-page/authority-organization-detail-page/authority-organization-detail-page.component.ts

```diff
@@ -192,6 +192,7 @@ export class AuthorityOrganizationDetailPageComponent
           icon: 'article',
           isDisabled: true, // no page associated yet
           value: organization.dataOfferCount,
+          tooltip: `Available: ${organization.liveDataOfferCount}\nOn Request: ${organization.onRequestDataOfferCount}`,
```

Comment:
missing angular's "formatNumber" for locale-correct thousand separators. probably here and in the table. in a template you can go 'x | formatNumber'

Reply:
I think this could be counter-productive. For example the organization detail has a limited amount of space for even 4 digits. Putting a period there would make it ugly. I also don't think that the numbers will ever reach sizes where thousands-separation would be necessary

Reply:
it's not really about the space, it's something that should be applied to any number shown in a frontend, but something you probably don't have on your radar yet. this is why I'd like to establish now the eye to add this by default to any number displayed. thousand separators are important